### PR TITLE
Removing /deploy-radiator and /deploys-notify admin endpoints

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
Removing /deploy-radiator and /deploys-notify admin endpoints

## What is the value of this and can you measure success?
With Prout and Continuous Deployment enabled those endpoints are not
required anymore
🔪 

cc @guardian/dotcom-platform

### CODE REVIEW

**Comment by sndrs** [comment]
looks like all the assets stuff is caught 👍

File: admin/conf/routes [hunk 1]
```diff
@@ -109,13 +109,7 @@ GET         /metrics/afg
...
 
-GET         /deploys                                                                                controllers.admin.DeploysRadiatorController.getCompletedDeploys(pageSize: Option[String], projectName: Option[String], stage: Option[String])
-
-GET         /deploys-radiator/api/deploys                                                           controllers.admin.DeploysRadiatorController.getDeploys(pageSize: Option[String], projectName: Option[String], stage: Option[String], status: Option[String])
-GET         /deploys-radiator/api/builds/:number                                                    controllers.admin.DeploysRadiatorController.getBuild(number)
-GET         /deploys-radiator                                                                       controllers.admin.DeploysRadiatorController.renderDeploysRadiator()
-
-POST        /deploys-notify/api/builds/:number/notify                                               controllers.admin.DeploysNotifyController.notifyStep(number)
+GET         /deploys                                                                                controllers.admin.DeploysController.getDeploys(stage: Option[String], pageSize: Option[Int])
```
Comment (jfsoul) [inline admin/conf/routes]: Can we put a comment in somewhere (here or controller) to say this is primarily used by Prout?

File: docs/01-start-here/03-how-to-deploy.md [hunk 2]
```diff
@@ -3,25 +3,25 @@
...
 1. Merge your pull request to master
-2. Wait for the tests and the build to finish (Teamcity)
-3. Once the build is done, it will be automatically deployed to CODE (The Guardian staging environment)
-4. Verify that your change works properly on CODE
-5. You can find out who is also currently deploying by using the [Deploy Radiator](https://frontend.gutools.co.uk/deploys-radiator) (Private)
-6. If several people are pushing to PROD at the same time, please coordinate with each other. Slack channel #dotcom-push is a good place to do so.
-7. When ready use [Riff-Raff to deploy](https://riffraff.gutools.co.uk/deployment/request) (Private) the `dotcom:all` artifact to production. The build to deploy can be found in Teamcity ([example](images/build-number-tc.png)) or on the [Deploy Radiator](https://frontend.gutools.co.uk/deploys-radiator) (Private)
-8. Verify that your changes are working in production and check the [logs](https://kibana.gu-web.net/goto/7c4f5f1b8b3c0b49055b3611d5d7810e) (Private) for any issue related to the deployment.
+1a. Wait for the tests and build to finish (Teamcity)
+3. Once the build step is completed, it will be automatically deployed to PROD.
```
Comment (jfsoul) [inline docs/01-start-here/03-how-to-deploy.md]: 2?

File: admin/conf/routes (hunk 1 above)
Comment (TBonnin) [inline admin/conf/routes]: yep. I will do that

File: docs/03-dev-howtos/13-Update-configuration-in-s3.md [hunk 3]
```diff
@@ -12,35 +12,35 @@ To add or update a configuration item you need to:
 aws s3 ls --profile=frontend s3://aws-frontend-store/config/
 ```
 
-look for the most recent version ( the `v14` part ):
+look for the most recent version ( the `v17` part ):
```
Comment (jfsoul) [inline docs/03-dev-howtos/13-Update-configuration-in-s3.md]: I'm not sure continuing to bump this every time is particularly valuable? Clearly, I never remember anyway! 🙈

Comment (TBonnin) [inline docs/03-dev-howtos/13-Update-configuration-in-s3.md]: It is definitely not required to bump those. 
I just find it easier when I have to update the config. I just have to copy paste without changing the number. 
Anyway the whole process is cumbersome. Hopefully the new devtools team will come up with a nice config tool :crossed_fingers:

**Review by jfsoul — APPROVED** [review]
Looks good to me once the markdown numbering is corrected!

So much code gone! ✂️

**Pull request merged**